main.scss

```diff
@@ -1,123 +1,107 @@
-@import
-"o-colors/main",
-"o-typography/main",
-"o-icons/main";
+@import "o-colors/main";
+@import "o-typography/main";
+@import "o-icons/main";
 
 @if ($o-cookie-message-is-silent == false) {
-	.o-cookie-message {
-		display: block;
-	}
+    /*------------------------------------*\
+            #BLOCK-DESKTOP
+    \*------------------------------------*/
+
+    .o-cookie-message {
+        position: relative;
+        background-color: #fff1de;
+        border-bottom: 1px solid #d0c6b8;
+        padding: 5px 15px;
+
+        &__container {
```

Comment:
we avoid using the ampersand for classes so it's easier to search in the code, and also for specificity, by using the full name, there's no need for nesting the class as it already has the `o-cookie-message` prefix